### PR TITLE
🐛  Fix create soft link kwargs

### PR DESCRIPTION
**Pull request by bisgaard-itis**
## What do these changes do?
- This PR fixes a bug discovered by @wvangeit.
- It was caused by the changes I have recently done to how the error handling is done in the api-server
- I also add a test to ensure the function is covered by our tests. Seems that was not previously the case. 

## Dev-ops checklist

- [x] No ENV changes or I properly updated ENV ([read the instruction](https://git.speag.com/oSparc/osparc-ops-deployment-configuration/-/blob/configs/README.md?ref_type=heads#how-to-update-env-variables))

### CODE REVIEW

**Review by wvangeit — APPROVED** [review]
🥳🥳🥳🥳

File: services/api-server/src/simcore_service_api_server/services/study_job_models_converters.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ async def create_job_outputs_from_project_outputs(
     job_id: StudyID,
     project_outputs: dict[NodeID, dict[str, Any]],
     user_id,
```
Comment (pcrespov) [inline services/api-server/src/simcore_service_api_server/services/study_job_models_converters.py:89]: I guess `mypy` would have capture this if the annotations were in place
Please annotate **all** the inputs, also `user_id`  :-)

**Review by pcrespov — APPROVED** [review]

**Review by GitHK — APPROVED** [review]

File: services/api-server/src/simcore_service_api_server/services/study_job_models_converters.py (hunk 1 above)
Comment (bisgaard-itis) [reply services/api-server/src/simcore_service_api_server/services/study_job_models_converters.py:89]: Ok, I will do that, but in a separate PR. This one should ideally go into staging today to unblock Werner. So I will leave it as is for now.

**Review by matusdrobuliak66 — APPROVED** [review]

**Comment by bisgaard-itis** [comment]
thanks @pcrespov

**Pull request merged**